Chutzpah/Models/ChutzpahTestSettingsFile.cs

```diff
@@ -396,6 +396,8 @@ public Regex TestPatternRegex
             }
         }
 
+        public int? IEBrowserTabOpenTimeout { get;  set; }

```

Comment:
Does this need to be a setting? I am fine with it but curious when someone would need to adjust.

Reply:
When I was first encountering this issue, I thought maybe the Chutzpah Test Adapter wasn't able to find the IE tab within the default 4 second timeout because I thought IE may have still been trying to load.

I could imagine this would be an issue on relatively slow machines, when other processes are fighting for resources, or if there are any IE add-ons that take some time to load with the browser.

It also seems like having a hard-coded timeout value isn't the best practice.

I can revert this change if you think it's best.

Thank you.

Reply:
Oh I am fine with it, just wanted to understand the rationale.